pkg/ignition/builder.go

```diff
@@ -73,6 +73,9 @@ func (b *ignitionBuilder) ProcessNetworkState() (error, string) {
 			}
 			return err, ""
 		}
+		if string(out) == "--- {}\n" {
+			return nil, "no network configuration file(s): " + string(out)
```

Comment:
No point to add string(out) to the message: it does not provide any details

Reply:
ack @dtantsur. I will make the change. And will the `no network configuration file(s)` be the appropriate message here?

Reply:
Hello @dtantsur, no `stderr`. Hence I have changed the message to `no network configuration`. Please review and let me know if anything needs to be added.